### ISSUE TITLE
[Feature Request] Delete movies and series

### ISSUE THREAD

**Issue by Eseltwift** [thomst08/requestrr#27]
Hi!

I think it would be really handy to be able to send a Radarr movie or Sonarr series deletion request directly via the Discord bot. We could then choose whether to delete the files on the storage space as well.
We could imagine a `/delete <movie|tv> <media name>` and set the files deletion choice on the web UI, or directly `/delete <option> <movie|tv> <media name>` (I think the first solution is easier to understand and implement).

In short, the Discord bot would send an API request equivalent to this:
![image](https://github.com/thomst08/requestrr/assets/98350888/62ab08b4-30df-4310-af69-e23dcfc87ca5)

**Comment by thomst08** [comment]
Hey @Eseltwift,

Thanks for reaching out, something similar has been suggested in the past and it's on my list of things to look at.
Your solution isn't a bad one, but it might be a while until I'm able to look at it.

Basically, great idea, I've added it to the list :)
